src/main/java/com/eerussianguy/blazemap/feature/maps/MapRenderer.java

```diff
@@ -501,7 +520,7 @@ public void setSearch(String search) {
             return;
         }
         try {
-            Pattern pattern = Pattern.compile(search);
+            Pattern pattern = Pattern.compile(search, Pattern.CASE_INSENSITIVE);
```

Comment:
This is good, but by itself isn't enough, you need to do the equivalent inside the catch for when the search term isn't valid regex

Reply:
Good catch! 

- [ ] Fix error handling for regex case insensitivity